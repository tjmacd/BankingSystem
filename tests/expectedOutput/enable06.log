10                      00000 00000000 A 
09 Disabled Account     00006 00000000   
01 Disabled Account     00006 00005.00   
02 Disabled Account     00006 00000.01   
02 Disabled Account     00001 00000.01   
03 Disabled Account     00006 00000.01 EC
04 Disabled Account     00006 00000.01   
08 Disabled Account     00006 00000000   
00                      00000 00000000   