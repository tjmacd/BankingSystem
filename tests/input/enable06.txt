login
admin
enable
Disabled Account
00006
withdrawal
Disabled Account
00006
5.00
transfer
Disabled Account
00006
00001
0.01
paybill
Disabled Account
00006
EC
0.01
deposit
Disabled Account
00006
0.01
changeplan
Disabled Account
00006
logout