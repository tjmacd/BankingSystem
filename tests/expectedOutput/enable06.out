Please select session type (standard or admin):
Logged in as admin
Please enter account holder's name:
Enter the account number:
Account 00006 enabled
Please enter account holder's name:
Enter the account number:
Enter the amount to withdraw:
$5.00 withdrawn from account 00006
Please enter account holder's name:
Enter the account number to transfer from:
Enter the account number to transfer to:
Enter the amount to transfer:
$0.01 transfered to account 00001
Please enter account holder's name:
Enter the account number:
Enter the payee company:
Enter the amount to pay:
$0.01 payed to EC
Please enter account holder's name:
Enter the account number:
Enter the amount to deposit:
$0.01 deposited to account 00006
Please enter account holder's name:
Enter the account number:
00006 has been changed to a student account
Logged out